src/index.js

```diff
@@ -155,17 +155,28 @@ export default class StepWizard extends PureComponent {
     render() {
         const props = {
             currentStep: this.state.activeStep + 1,
-            totalSteps: this.props.children.length,
+            totalSteps: this.props.children.filter(el => el).length,
             /** Functions */
             nextStep: this.nextStep,
             previousStep: this.previousStep,
             goToStep: this.goToStep,
             firstStep: this.firstStep,
             lastStep: this.lastStep,
         };
+        /**
+         * from code by https://github.com/wallawe
+         * found on https://github.com/jcmcneal/react-step-wizard/issues/55
+         */
+        const childrenMinusNulls = [];
+
+        React.Children.forEach(this.props.children, (child) => {
+            if (child) {
+                childrenMinusNulls.push(child);
+            }
+        });
 
         const { classes } = this.state;
-        const childrenWithProps = React.Children.map(this.props.children, (child, i) => {
+        const childrenWithProps = React.Children.map(childrenMinusNulls, (child, i) => {
```

Comment:
Why not just check if the child is `undefined` here, as the first condition, and if so then return `null`? Then you won't need the extra fluff and loop above.

Reply:
Hello, I commited a improved way based on your comment.

Reply:
Thanks for making these changes